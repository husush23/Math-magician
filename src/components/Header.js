/*eslint-disable*/
import React from 'react';
import {Link} from 'react-router-dom';

function Header() {
  return (
    <header className='header'>
      <h1>Math Magcian</h1>
      <nav>
        <Link to='/'>Home</Link>
        <Link to='/calculator'>Calculator</Link>
        <Link to='/quotes'>Quotes</Link>
      </nav>
    </header>
  );
}

export default Header;
