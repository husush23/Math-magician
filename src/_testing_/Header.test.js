import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../components/Header';
import { BrowserRouter } from 'react-router-dom';


test('renders correctly', () => {
  const tree = renderer.create(
  <BrowserRouter>
    <Header />
  </BrowserRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
})