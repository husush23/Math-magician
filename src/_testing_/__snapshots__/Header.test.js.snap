// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="header"
>
  <h1>
    Math Magcian
  </h1>
  <nav>
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      href="/quotes"
      onClick={[Function]}
    >
      Quotes
    </a>
  </nav>
</header>
`;
